pkg/client/unversioned/remotecommand/resize/resize.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resize
+
+import (
+	"encoding/json"
+	"io"
+
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/kubernetes/pkg/util/term"
```

Comment:
This way `term` is still depended by the package `remotecommand`. Can we move the definition of [Size](https://github.com/kubernetes/kubernetes/blob/master/pkg/util/term/resize.go#L26-L30) and [TerminalSizeQueue](https://github.com/kubernetes/kubernetes/blob/master/pkg/util/term/resize.go#L75-L79) to the package `remotecommand` to reverse the dependency direction?